src/Components/CreateImageWizard/ValidatedInput.tsx

```diff
@@ -85,6 +92,20 @@ export const HookPasswordValidatedInput = ({
           </Button>
         </InputGroupItem>
       </InputGroup>
+      {validated === 'error' && (
+        <HelperText>
+          <HelperTextItem variant="error" hasIcon>
+            {stepValidation.errors[fieldName]}
+          </HelperTextItem>
+        </HelperText>
+      )}
+      {warning !== undefined && warning !== '' && (
+        <HelperText>
+          <HelperTextItem variant="warning" hasIcon>
+            {warning}
+          </HelperTextItem>
+        </HelperText>
+      )}
```

Comment:
Umm, there is already validation implemented in the `HookValidatedInput`. This just seems to be duplicating a bunch of code. Couldn't you just add the password validation to the `useUsersValidation`?

Try to take a look at `useValidation` changes here: https://github.com/osbuild/image-builder-frontend/pull/2837/files#diff-e8f80e37ebfb45fc960d754061f3cd5877b1110ca14915bdcc1408571a04dad2

Reply:
I tried to implemented it in the `HookValidatedInput` and the error appears at the right side intead of under, I will push again this implement in different commit
<img width="1371" alt="Screenshot 2025-02-09 at 15 14 17" src="https://github.com/user-attachments/assets/838c5b98-9caf-4a3a-9320-312c5c48de1c" />

Reply:
I implemened it diffrent, thank you